ush/forecast_postdet.sh

```diff
@@ -500,6 +500,7 @@ FV3_nml(){
 
 FV3_out() {
   echo "SUB ${FUNCNAME[0]}: copying output data for FV3"
+  mkdir -p "${COM_CONF}"
```

Comment:
`common_predet` is probably a better place for this.

Reply:
Done.